### PR TITLE
Update Keytar to v6.0.1

### LINKED ISSUE #28: Support Node.js 14
The Keytar dependency must be updated to v6 to support Node.js 14, which will become the LTS version of Node.js on 2020-10-27.

### PR DESCRIPTION
**Pull request by ghost**
Resolves #28

### CODE REVIEW

**Review by zFernand0 — DISMISSED** [review]
Seems good to me.
We should keep an eye out for whenever they drop support for Node 8 (Node Module Version v57)
https://github.com/atom/node-keytar/releases

File: jenkins/bundleKeytar.sh [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+cd "$(git rev-parse --show-toplevel)"
 
+rm -rf prebuilds
 mkdir prebuilds && cd prebuilds
+
 curl -fsL -o jq https://github.com/stedolan/jq/releases/latest/download/jq-linux64
 chmod +x ./jq
 
-curl -fs https://$1@api.github.com/repos/atom/node-keytar/releases/tags/v$2 |
+curl -fs https://$2@api.github.com/repos/atom/node-keytar/releases/tags/v$1 |
```
Comment (zFernand0) [inline jenkins/bundleKeytar.sh:13]: Curious if you could name them, no big deal though.

**Review by zFernand0 — COMMENTED** [review]
this PR still LGTM!

**Comment by zFernand0** [comment]
Thanks for updating gho... @t1m0thyj

**Review by zFernand0 — APPROVED** [review]

**Review by awharn — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**